.travis.yml

```diff
@@ -16,6 +16,8 @@ addons:
 services:
   - mysql  # https://docs.travis-ci.com/user/database-setup/#mysql
 
+before_install: phpenv config-add myconfig.ini
```

Comment:
Shouldn't this be added in the file from line 29?

Reply:
Yeah. Fixed